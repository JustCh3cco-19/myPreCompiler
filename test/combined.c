#include "combined.h"
// rimuovere questo commento
float x; double two&four;
char c = 'a'; /* block comment */
