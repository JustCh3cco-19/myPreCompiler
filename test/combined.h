#define GOOD   1
#define _X     2
#define BAD-NAME 3
