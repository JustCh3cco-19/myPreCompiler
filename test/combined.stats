combined.c:3: invalid identifier "BAD-NAME"
combined.c:4: invalid identifier "two&four"
Variables checked:   4
Invalid identifiers: 2
Comments removed:    2
Files included:      1
Input:  4 lines,  101 bytes
Output: 3 lines,  82 bytes
