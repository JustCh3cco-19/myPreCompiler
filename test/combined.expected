#define GOOD   1
#define _X     2
#define BAD-NAME 3
float x; double two&four;
char c = 'a';
